Workbooks/HDInsight/Workload-Specific/Kafka/settings.json

```diff
@@ -3,5 +3,5 @@
     "name":"HDInsight Kafka Insights",
     "author": "Microsoft",
     "isPreview": true,
-    "galleries": [{ "type": "workbook", "resourceType": "Azure Monitor", "order": 100 }]
```

Comment:
this change is removing this template from azure monitor all up, is that intentional?  it is possible for a template to be in both places, galleries is an array of entries.

Reply:
We actually do not plan on adding a "Workbooks" ToC blade to our cluster portal just yet, so should I just revert this change and only show them in the Azure Monitor section?